airflow/providers/datadog/hooks/datadog.py

```diff
@@ -115,8 +120,16 @@ def query_metric(self,
         return response
 
     # pylint: disable=too-many-arguments
-    def post_event(self, title, text, aggregation_key=None, alert_type=None, date_happened=None,
-                   handle=None, priority=None, related_event_id=None, tags=None, device_name=None):
+    def post_event(self, title: str,
+                   text: str,
+                   aggregation_key: Optional[str] = None,
+                   alert_type: Optional[str] = None,
+                   date_happened: Optional[int] = None,
+                   handle: Optional[str] = None,
+                   priority: Optional[str] = None,
+                   related_event_id: Optional[str] = None,
+                   tags: Optional[List[str]] = None,
+                   device_name: Optional[List[str]] = None) -> Dict[str, Any]:
```

Comment:
@mik-laj @potiuk @OmairK the `post_event` method returns a response. is `Dict` the proper type for this?